modules/sandboxed-containers-triggering-installation-kata-runtime.adoc

```diff
@@ -0,0 +1,7 @@
+//Module included in the following assemblies:
+//
+// * sandboxed_containers/deploying_sandboxed_containers.adoc
+
+[id="sandboxed-containers-triggering-installation-kata-runtime_{context}"]
+
+= Triggering the installation of the Kata runtime
```

Comment:
"Kata runtime" here, but "Kata Runtime" in sandboxed-containers-uninstalling-kata-runtime.adoc (I think this one is correct?)

Reply:
For posterity: It should be 'the Kata runtime'.